### DISCUSSION TITLE
Bosean FS-5000: Display improvement

### DISCUSSION THREAD

**Discussion by Ni5ei** [Gissio/radpro#131] [Ideas]
Thanks for making RAD Pro available on the Bosean FS-5000
I really like the look and functionality but there's a lot of empty space on the main screen which can be used to display more information.
Besides the main units chosen, it could easily display different units in a smaller font below that. I really would like to use μSv/h as my main unit but still be able to see cps and cpm underneath it. Having to go back and forth in the menu to change units is an option but you never get to see the units next to eachother. There's space enough so why not make use of it?

**Comment by Gissio** [comment]
Built with simplicity in mind, Rad Pro adheres to the [KISS principle](https://en.wikipedia.org/wiki/KISS_principle).

However, you can display two units by changing the secondary view with the ok/power key.

**Reply by Ni5ei** [reply]
Thanks, I missed that.
